### PR TITLE
[zh-cn]: translate for CookieChangeEvent constructor

### PR DESCRIPTION
**Pull request by xyy94813**
### Description

translate CookieChangeEvent constructor docs to zh-cn

### Motivation

learning and translating

### Additional details

None.

### Related issues and pull requests

None.

### CODE REVIEW

File: files/zh-cn/web/api/cookiechangeevent/cookiechangeevent/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+new CookieChangeEvent(type)
+new CookieChangeEvent(type, options)
+```
+
+### 参数
+
+- `type`
+  - : 写有此事件名称的字符串。大小写敏感且浏览器经常设置为 `change`。
+- `options` {{Optional_Inline}}
+  - : 一个对象，该对象除了 Event（）中定义的属性之外，还可以具有以下属性：
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/cookiechangeevent/cookiechangeevent/index.md:26]:
```suggestion
  - : 一个对象，该对象除了在 {{domxref("Event/Event", "Event()")}} 中定义的属性之外，还可以具有以下属性：
```

File: files/zh-cn/web/api/cookiechangeevent/cookiechangeevent/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - `changed`{{Optional_Inline}}
+      - : 返回包含已被修改的 cookie 的数组。
+    - `deleted`{{Optional_Inline}}
+      - : 返回包含已被删除的 cookie 的数组。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/cookiechangeevent/cookiechangeevent/index.md:30]:
```suggestion
    - `changed`{{Optional_Inline}}
      - : 一个包含已被修改的 cookie 的数组。
    - `deleted`{{Optional_Inline}}
      - : 一个包含已被删除的 cookie 的数组。
```

File: files/zh-cn/web/api/cookiechangeevent/cookiechangeevent/index.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+```js-nolint
+new CookieChangeEvent(type)
+new CookieChangeEvent(type, options)
+```
+
+### 参数
+
+- `type`
+  - : 写有此事件名称的字符串。大小写敏感且浏览器经常设置为 `change`。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/cookiechangeevent/cookiechangeevent/index.md:24]:
```suggestion
  - : 包含事件名称的字符串。它区分大小写，且浏览器总是将其设置为 `change`。
```

File: files/zh-cn/web/api/cookiechangeevent/cookiechangeevent/index.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+- `options` {{Optional_Inline}}
+  - : 一个对象，该对象除了在 {{domxref("Event/Event", "Event()")}} 中定义的属性之外，还可以具有以下属性：
```
Comment (jasonren0403) [inline files/zh-cn/web/api/cookiechangeevent/cookiechangeevent/index.md:26]:
```suggestion
  - : 一个对象，该对象*除了在 {{domxref("Event/Event", "Event()")}} 中定义的属性之外*，还可以具有以下属性：
```

**Review by jasonren0403 — COMMENTED** [review]
一些改进建议

**Review by jasonren0403 — APPROVED** [review]
👍 感谢 @xyy94813 和 @skyclouds2001

**Pull request merged**